SortProject/SortProject/Sort.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+
+namespace Sort
+{
+    public class SetSort
+    {
+        private int count = 0;
+
+        // Создание свойства только для чтения
+        public int Count
+        {
+            get { return count; }
+        }
+
+        // Пустой список, с которым в дальнейшим будут вестись все манипуляции методов класса
+        public int[]? myset;
```

Comment:
nit:
```suggestion
        public int[] myset;
```
поле объявляется, а инициализируется в конструкторе.
в таком случает nullability тут не нужно, уменьшит число проверок на null по коду